src/components/casestudyandcoretools/HightlightCaseStudyDetails.tsx

```diff
@@ -0,0 +1,61 @@
+import React from "react";
+import ArrowDown from "../../components/icons/ArrowDown";
+
+const caseStudyLevel = [
+  "Household",
+  "Community",
+  "Policy/Systemic",
+  "Productivity Use",
+];
+const caseStudyTechnology = [
+  "Coding",
+  "Lighting",
+  "Hummanitarian Inflastructure",
+];
+
+const HightlightCaseStudyDetails = () => {
+  return (
+    <div className="flex items-center flex-col">
+      <ArrowDown />
+      <h1 className="text-center text-gray-500 text-xl font-bold mb-3 ">
+        Hightlight case study details
+      </h1>
+      <div className="bg-gray-200 rounded-2xl flex gap-2 py-4 px-8 border border-gray-500">
+        <div className="p-4">
+          <h1 className="text-center text-gray-500 text-xl font-bold mb-3 ">
+            Case study imprementation level
+          </h1>
+          <div>
+            <div className="flex flex-col gap-2 p-4">
+              {caseStudyLevel.map((el, index) => (
```

Comment:
Could you use more understandable name instead of "el" ex:element or item

Reply:
fixed

Reply:
fixed